### ISSUE TITLE
Exact prop does not work on d-nav-item

### ISSUE THREAD

**Issue by simioluwatomi** [DesignRevision/shards-vue#10]
## Expected Behavior
The ```d-nav-item``` with an ```:exact``` prop should not have an active class when it is not the current active route

## Current Behavior
Even after applying the ```active``` prop to a ```d-nav-item```, an active class is always applied to it as long as the current path is preceeded by the path of the ```d-nav-item``` with the ```exact``` prop.

```
<d-collapse is-nav id="nav-collapse" class="justify-content-md-center">
    <d-navbar-nav>
        <d-nav-item to="/" exact>Home</d-nav-item>
        <d-nav-item to="/portfolio">Portfolio</d-nav-item>
        <d-nav-item to="/stocks">Stocks</d-nav-item>
        <d-nav-item to="">End Day</d-nav-item>
        <d-dropdown text="Save & Load" is-nav>
            <d-dropdown-item>Action</d-dropdown-item>
            <d-dropdown-item>Another action</d-dropdown-item>
            <d-dropdown-item>Something else here</d-dropdown-item>
            <d-dropdown-divider />
            <d-dropdown-item>Separated link</d-dropdown-item>
        </d-dropdown>
    </d-navbar-nav>
</d-collapse>
```
```
export default new Router({
    linkActiveClass: 'active',
    linkExactActiveClass: 'exact-active',
   // more router options
})

```

Result

![untitled](https://user-images.githubusercontent.com/22191563/47391895-f1beda00-d712-11e8-9f4b-ca97c3e66918.png)

**Comment by simioluwatomi** [comment]
@hisk you don't need to apologise for anything... I'll be waiting for that fix!!!!

**Comment by hisk** [comment]
@simioluwatomi This issue has been fixed in 1.0.5. Thanks for your patience! 😄
